manim/mobject/mobject.py

```diff
@@ -1721,33 +1750,39 @@ def length_over_dim(self, dim):
         ) - self.reduce_across_dimension(np.min, np.min, dim)
 
     def get_coord(self, dim, direction=ORIGIN):
-        """Meant to generalize get_x, get_y, get_z"""
+        """Meant to generalize ``get_x``, ``get_y`` and ``get_z``"""
         return self.get_extremum_along_dim(dim=dim, key=direction[dim])
 
     def get_x(self, direction=ORIGIN):
+        """Returns x coordinate of the center of the mobject as ``numpy.float64`` """
         return self.get_coord(0, direction)
 
     def get_y(self, direction=ORIGIN):
+        """Returns y coordinate of the center of the mobject as ``numpy.float64`` """
         return self.get_coord(1, direction)
 
     def get_z(self, direction=ORIGIN):
+        """Returns z coordinate of the center of the mobject as ``numpy.float64`` """
         return self.get_coord(2, direction)
 
     def get_start(self):
+        """Returns the point, where the stroke that surrounds the mobject starts.  Type is a ``numpy.ndarray`` with 3 elements, e.g. [-1,0,0] """
```

Comment:
Should the part " Type is a ``numpy.ndarray`` with 3 elements, e.g. [-1,0,0] " be removed?
I think this is a bit long

Reply:
Yes. It should be under a Return section and should also use type hints.

Also this should be done for the other methods you've documented as well (but particularly get_end, and get_start_and_end)

Reply:
okay!
```suggestion
        """Returns the point, where the stroke that surrounds the :class:`~.Mobject` starts. 
```